airbyte-workers/src/test/java/io/airbyte/workers/DefaultSyncWorkerTest.java

```diff
@@ -104,4 +113,37 @@ void test() throws Exception {
     verify(target).close();
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
hehe this tests caught a bunch of information we were failing to populate in the sync summary.